src/lib/selections.ts

```diff
@@ -549,9 +571,10 @@ export function getSelectionTypeDisplayText(
     .map(
       // Hack for showing "face" instead of "extrude-wall" in command bar text
       ([type, count]) =>
-        `${count} ${type.replace('extrude-wall', 'face')}${
-          count > 1 ? 's' : ''
-        }`
+        `${count} ${type
+          .replace('wall', 'face')
+          .replace('solid2D', 'face')
+          .replace('segment', 'face')}${count > 1 ? 's' : ''}`
```

Comment:
This probably made sense when the main thing was `extrude-wall`, this might need to be neatend up.

Reply:
Yeah I think you're right. I can give it a think and contribute if you like

Reply:
Sounds good.